modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -78,64 +78,89 @@ const assertStructureHasCustomStyle = (editor: Editor, rows: number, columns: nu
     '</table>');
 };
 
-const closeMenu = (editor) =>
+const closeMenu = (editor) => {
   TinyUiActions.keydown(editor, Keys.escape());
+  TinyUiActions.keydown(editor, Keys.escape());
```

Comment:
Is this needed?

Reply:
Yes, when you open the menuitems it requires an extra escape.